target/linux/zynq/config-4.14

```diff
@@ -0,0 +1,752 @@
+CONFIG_ADVISE_SYSCALLS=y
```

Comment:
would it be an idea to squash this with `zynq: remove support for kernel 4.4` ? and just do "zynq: move kernel config to 4.14" ?
it would be one less commit; especially if this is a copy

Reply:
Yes, I did that initially, but decided to split the changes into two commits so the removal could be reverted in case someone needs the 4.4 kernel in the future.

I can squash the changes if you still think is appropriate

Reply:
not sure;
let's see if anyone else has any other thoughts; [someone with merge rights]
otherwise you implement the change, and he/she may ask you to leave it like this;